contrib/apelscripts/condor_batch.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+
+# Create a temporary accounting file name
+today=$(date -u --date='00:00:00 today' +%s)
+yesterday=$(date -u --date='00:00:00 yesterday' +%s)
+output=batch-$(date -u --date='yesterday' +%Y%m%d )-$(hostname -s)
+
+# Build the filter for the history command
+CONSTR="EnteredCurrentStatus >= $yesterday && EnteredCurrentStatus < $today && RemoteWallclockTime !=0"
+
+function die() { echo "$0 aborted when reading config"; exit 1; }
+trap die ERR;
+if [ $# -gt 0 ]; then
+  source $1
+fi
+trap '' ERR
+
+[[ -z "$SCALING_ATTR" ]] && SCALING_ATTR=MachineAttrRalScaling0
```

Comment:
The `[[ ... ]]` syntax is only available in bash; you should change the shebang line to `#!/bin/bash` instead.